Assets/Students/HenryLiao/BackAndForthMove.cs

```diff
@@ -1,30 +0,0 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class BackAndForthMove : MonoBehaviour
```

Comment:
Reuse the ObstaclePath script. It may need some fixing. Feel free to rename functions / classes to make this more generic.